exporter/kineticaexporter/factory.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// NewFactory creates a factory for Kinetica exporter.
+//
+//	@return exporter.Factory
+func NewFactory() exporter.Factory {
+	return exporter.NewFactory(
+		typeStr,
+		CreateDefaultConfig,
+		exporter.WithLogs(createLogsExporter, component.StabilityLevelDevelopment),
+		exporter.WithTraces(createTracesExporter, component.StabilityLevelDevelopment),
+		exporter.WithMetrics(createMetricsExporter, component.StabilityLevelDevelopment),
+	)
+}
+
+// CreateDefaultConfig - function to create a default configuration
+//
+//	@return component.Config
+func CreateDefaultConfig() component.Config {
+	return &Config{
+		Host:               "http://localhost:9191/",
```

Comment:
Is it a common use case to run the database locally?  Would it be better to default this to `""` and validate that a value is supplied by the user?

Reply:
There are cases where the database could be local; so we think we could keep this as a valid default.